test_conformance/math_brute_force/main.cpp

```diff
@@ -1201,43 +1144,6 @@ test_status InitCL( cl_device_id device )
         return TEST_FAIL;
 #endif
     }
-#if defined( __APPLE__ )
-    else if(is_extension_available(gDevice, "cl_APPLE_fp64_basic_ops" ))
```

Comment:
Do recent Apple devices support cl_khr_fp64? If they don't we may want to raise an issue to modify the double path to support this as well but I agree the current implementation needs to go.

Reply:
The ones in my system do. But I don't think we should be testing vendor extensions in CTS anyway?

Reply:
Ok, I think that settles it for me. I suggest we don't create an issue for now. It's likely an old extension that pre-dates cl_khr_fp64.

On vendor extensions, we did discuss it at a previous F2F (I think it was the last one) and concluded that we'd make decisions on a case-by-case basis for a number of reasons:
- This came up as a requirement to allowing negative testing (a vendor might have an extension to relax an error case, they would need to add support to the CTS).
- There is value in having testing for vendor extensions when it's not intrusive as it makes it possible for other vendors to evaluate the feature and users to check conformance of their platform and could encourage promotion to ext/khr.
- There are cases where the CTS relies on implementation-defined behaviour. This is the reason we need support for cl_arm_printf for example.

Reply:
Makes sense, thanks for the info.